website/source/api/agent/service.html.md

```diff
@@ -58,6 +58,66 @@ $ curl \
 }
 ```
 
+## Get local service health
+
+The `/v1/agent/health/service/name/<service_name>` and
```

Comment:
nit (wording):
  "allow to retrieve" = "allow retrieving"

Reply:
@mkeeler DONE